File: build-tools/codeassist-builder-plugin/src/main/java/com/tyron/builder/model/VectorDrawablesOptions.java
package com.tyron.builder.model;

import org.jetbrains.annotations.Nullable;

import java.util.Set;

/**
 * Options for build-time support for vector drawables.
 */
public interface VectorDrawablesOptions {

    @Nullable
    Set<String> getGeneratedDensities();

    @Nullable
    Boolean getUseSupportLibrary();
}

File: build-tools/codeassist-builder-plugin/src/main/java/com/tyron/builder/model/v2/ide/VectorDrawablesOptions.kt
package com.tyron.builder.model.v2.ide

import com.tyron.builder.model.v2.AndroidModel

/**
 * Options for build-time support for vector drawables.
 *
 * @since 4.2
 */
interface VectorDrawablesOptions: AndroidModel {
    val generatedDensities: Set<String>?
    val useSupportLibrary: Boolean?
}